README.md

```diff
@@ -4,16 +4,42 @@ Language server for Ralph.
 
 Currently supports text document level events and diagnostics.
 
+# Installation
```

Comment:
Feel free to ignore this: WDYT of moving this step after `Install VSCode plugin` or within `Install Neovim plugin`? Most devs probably just want to download plugins for their IDEs, and this installation step is not required for VSCode but is required for Neovim.

Reply:
I'll move it inside neovim section for now